mdn/tests/test_views.py

```diff
@@ -236,8 +305,51 @@ def test_get_without_issues(self):
         self.assertEqual(0, response.context_data['count'])
 
 
+class TestIssuesDetailCSV(TestCase):
+    def test_get(self):
+        self.feature = self.create(Feature, slug='web-css-float')
+        self.fp = FeaturePage.objects.create(
+            url="https://developer.mozilla.org/en-US/docs/Web/CSS/float",
+            feature_id=self.feature.id, data='{"foo": "bar"}',
+            status=FeaturePage.STATUS_PARSED)
+        self.issue = Issue.objects.create(
+            page=self.fp, slug="inline-text", start=10, end=20,
+            params='{"text": "inline"}')
+        url = reverse('issues_detail_csv', kwargs={'slug': 'inline-text'})
+        response = self.client.get(url)
+        self.assertEqual(200, response.status_code)
+        csv_lines = response.content.decode('utf8').splitlines()
+        url = 'http://testserver/importer/{}'.format(self.fp.pk)
+        expected = [
+            'MDN Slug,Import URL,Source Start,Source End,text',
+            'docs/Web/CSS/float,{},10,20,inline'.format(url),
+        ]
+        self.assertEqual(expected, csv_lines)
+
+
 class TestIssuesSummary(TestCase):
     def test_get(self):
         url = reverse('issues_summary')
         response = self.client.get(url)
         self.assertEqual(200, response.status_code)
+
+
+class TestIssuesSummaryCSV(TestCase):
+    def test_get(self):
+        self.feature = self.create(Feature, slug='web-css-float')
+        self.fp = FeaturePage.objects.create(
+            url="https://developer.mozilla.org/en-US/docs/Web/CSS/float",
+            feature_id=self.feature.id, data='{"foo": "bar"}',
+            status=FeaturePage.STATUS_PARSED)
+        self.issue = Issue.objects.create(
+            page=self.fp, slug="inline-text", start=10, end=20,
+            params='{"text": "inline"}')
+        url = reverse('issues_summary_csv')
+        response = self.client.get(url)
+        self.assertEqual(200, response.status_code)
+        csv_lines = response.content.decode('utf8').splitlines()
+        expected = [
+            'Count,Issue',
+            '1,inline-text',
+        ]
+        self.assertEqual(expected, csv_lines)
```

Comment:
No more are planned.  All of the mdn is supposed to be throw away code!  Still, did the refactor.  This is my life.